cli/jhipster.js

```diff
@@ -35,24 +35,33 @@ if (!semver.satisfies(currentNodeVersion, minimumNodeVersion)) {
     process.exit(1);
 }
 
-requireCLI();
+let preferGlobal = false;
+if (process.argv.length >= 3) {
```

Comment:
I think since we use commander it already would have all arguments passed so may be you could use the commander api for this. Im on my phone so i cant point to the exact api now